### ISSUE TITLE
landmark-complementary-is-top-level docs' failing example doesn't fail

### ISSUE THREAD

**Issue by dbjorge** [dequelabs/axe-core#4949]
### Product

axe-core

### Description

The [`landmark-complementary-is-top-level` doc page](https://dequeuniversity.com/rules/axe/4.11/landmark-complementary-is-top-level) includes this failing example:

> The following example FAILS the Complementary landmarks are at the top level rule:
>
> ```html
> <main>
>   <p>Some text</p>
>   <aside><p>An aside</p></aside>
> </main>
> ```

However, that example doesn't actually fail the rule - we started passing it when #2651 was fixed. We should replace the example with something that actually fails the rule.
